tests/integration/backward_compatible/proxy/cp/atomic_long_test.py

```diff
@@ -106,30 +106,30 @@ def test_set(self):
         self.assertIsNone(self.atomic_long.set(42))
         self.assertEqual(42, self.atomic_long.get())
 
-    @set_attr(min_server_version="4.1")
```

Comment:
Was the backward compatibility test runner supposed to check this attribute?

Reply:
I changed the runner so that it doesn't check it anymore. We dynamically skip tests at runtime now, just like the Node.js client

Reply:
Just like I predicted, good 👍🏻